lib/Backends/OpenCL/OpenCL.h

```diff
@@ -215,6 +215,23 @@ class OCLBackend final : public BackendUsingGlowIR {
   /// @}
 };
 
+namespace runtime {
+struct OpenCLDeviceBindings : DeviceBindings {
+  OpenCLDeviceBindings() : DeviceBindings(BackendKind::OpenCL) {}
+
+  /// CL memory buffer.
```

Comment:
It could be interesting to describe which of these fields are run-specific and which are network specific.